### PR TITLE
[YUNIKORN-2354] Detailed queue info 

### PR DESCRIPTION
**Pull request by doupache**
### What is this PR for?
queue-info

https://github.com/user-attachments/assets/11365d21-50bc-4003-938c-6020a136d8e8

### What type of PR is it?
* [x] - Feature

### What is the Jira issue?
[YUNIKORN-2354](https://issues.apache.org/jira/browse/YUNIKORN-2354)

### CODE REVIEW

File: src/app/utils/common.util.ts [hunk 1]
```diff
@@ -81,8 +81,42 @@ export class CommonUtil {
...
-    return value.split(', ').join('<br/>');
+  static absoluteUsedMemoryColumnFormatter(value: string | undefined): string {
+    if (!value) {
+      return '';
+    }
+    if (value === 'n/a') {
+      return '<strong>Memory:</strong> n/a';
+    }
+    let memory = value.split('%')[0] + '%';
+    console.log('mem', memory);
```
Comment (chenyulin0719) [inline src/app/utils/common.util.ts:92]: The console log should be removed.

**Review by chenyulin0719 — CHANGES_REQUESTED** [review]
@doupache Could you also add tests for below functions?

- absoluteUsedMemoryColumnFormatter()
- absoluteUsedCPUColumnFormatter()
- resourceColumnFormatter()

**Comment by doupache** [comment]
Thanks @chenyulin0719 for reviewing!

I have added the formatter tests and also removed the unnecessary console logs. 
Also added the missing guarantee block.

**Review by chenyulin0719 — CHANGES_REQUESTED** [review]
Hi @doupache,

Thanks for the update.
I found that this PR breaks the resource format on the application page. Could you check it?

image

**Comment by doupache** [comment]
I duplicated the `ResourceColumnFormatter`, creating one version for the queue page and another for the application page. I believe this duplication is acceptable and makes the code more readable.

image

**Comment by doupache** [comment]
### Inactive queue 
image

### Active queue
image


@craigcondit @chenyulin0719  I  have made layout tighter and add new queue data to `json.db`. 
PTLA

**Review by ryankert01 — APPROVED** [review]
+1, it's compact now! nice.

**Review by craigcondit — APPROVED** [review]
+1 LGTM.

**Pull request closed without merging**